### ISSUE TITLE
not building on openjdk6

### ISSUE THREAD

**Issue by tcurdt** [tcurdt/jdeb#224]
Using

```
    <mavenVersion>3.3.9</mavenVersion>
```

instead of 

```
    <mavenVersion>3.2.5</mavenVersion>
```

gives

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-plugin-plugin:3.4:descriptor (default-descriptor) on project jdeb: The API of the mojo scanner is not compatible with this plugin version. Please check the plugin dependencies configured in the POM and ensure the versions match. org/apache/maven/plugin/AbstractMojo : Unsupported major.minor version 51.0 -> [Help 1]
```

**Comment by dheid** [comment]
This simply means, that the Java 6 VM cannot run a class file that is compiled using the Java 7 compiler. Since both, Java 6 and Java 7, are no longer supported by Oracle, I would recommend to switch to Java 8.

In addition: For compatibility reasons I would not recommend to update to Maven 3.3.9, since currently used long term versions of Ubuntu (e.g. 14.04) are shipped with Maven 3.0.5.

**Comment by tcurdt** [comment]
not really worth keeping open ... java 8 it is
